nipype/interfaces/ants/registration.py

```diff
@@ -26,6 +26,26 @@ class ANTSInputSpec(ANTSCommandInputSpec):
                                   desc=('image to apply transformation to (generally a coregistered '
                                         'functional)'))
 
+#    Not all metrics are appropriate for all modalities. Also, not all metrics
+#    are efficeint or appropriate at all resolution levels, Some metrics perform
+#    well for gross global registraiton, but do poorly for small changes (i.e.
+#    Mattes), and some metrics do well for small changes but don't work well for
+#    gross level changes (i.e. 'CC').
+#
+#    This is a two stage registration. in the first stage
+#      [ 'Mattes', .................]
+#         ^^^^^^ <- First stage
+#    Do a unimodal registration of the first elements of the fixed/moving input
+#    list use the"CC" as the metric.
+#
+#    In the second stage
+#      [ ....., ['Mattes','CC'] ]
+#               ^^^^^^^^^^^^^^^ <- Second stage
+#    Do a multi-modal registration where the first elements of fixed/moving
+#    input list use 'CC' metric and that is added to 'Mattes' metric result of
+#    the second elements of the fixed/moving input.
```

Comment:
No worries. I'm sure all nipype users appreciate your contributions. I'll merge it the way it is. Thanks!